doc/api/cli.md

```diff
@@ -296,6 +296,30 @@ are linked from more than one location in the dependency tree (Node.js would
 see those as two separate modules and would attempt to load the module multiple
 times, causing an exception to be thrown).
 
+Note also that the `--preserve-symlinks` flag does not apply to the main
+module - this special case helps `node --preserve-symlinks node_module/.bin/<foo>`
```

Comment:
We wrap lines after 80 characters, otherwise, there will be a doc linting issue.